src/main/resources/META-INF/sql/herddb-schema.sql

```diff
@@ -113,3 +113,16 @@ CREATE TABLE IF NOT EXISTS tokens (
   description varchar(128),
   token varchar(1024)
 );
+
+CREATE TABLE IF NOT EXISTS users (
+  user_id BIGINT PRIMARY KEY AUTO_INCREMENT,
+  access_token varchar(256) NOT NULL,
+  name varchar(256) NOT NULL,
+  description varchar(128),
+  email varchar(256),
+  phone_number varchar(48),
+  location varchar(256),
+  company varchar(256),
+  expire LONG NOT NULL,
+  UNIQUE (name)
```

Comment:
Currently herddb does not implement UNIQUE constraint.
It does not report an error, I will file an issue.
It is better to drop this line

Reply:
Thank you, I will fix it